src/NServiceBus.RavenDB/Internal/DocumentStoreInitializer.cs

```diff
@@ -38,7 +38,9 @@ internal IDocumentStore Init(ReadOnlySettings settings)
 
         void ApplyConventions(ReadOnlySettings settings)
         {
-            var idConventions = new DocumentIdConventions(docStore, settings.GetAvailableTypes(), settings.EndpointName());
+            var sagasEnabled = settings.GetOrDefault<bool>(typeof(Features.Sagas).FullName);
```

Comment:
True. Can you make a feature that depends upon 2 other features being **disabled**?